src/Web.Api/appsettings.json

```diff
@@ -20,6 +20,7 @@
     "Enable": "true"
   },
   "SendGridApiKey": "",
+  "SendGridWebhookSignatureKey": "",
```

Comment:
Do not forget about build pipeline yml file - it also should be adjusted to get secret signature from GitHub secrets